src/main/java/htsjdk/samtools/SAMTag.java

```diff
@@ -64,20 +77,56 @@
     OC,
     OF,
     OR,
+    OX,
     PG,
     PQ,
     PT,
     PU,
     QT,
     Q2,
+    QX,
     R2,
     RG,
+    /**
+     * @deprecated use BC instead, for backwards compatibilty only
+     */
+    @Deprecated
     RT,
+    RX,
+    /** @deprecated for backwards compatibility only */
+    @Deprecated
     S2, // for backwards compatibility
     SA,
     SM,
+    /** @deprecated  for backwards compatibility only */
+    @Deprecated
     SQ, // for backwards compatibility
     TC,
     U2,
-    UQ
+    UQ;
+
+    private final short shortValue = SAMTag.makeBinaryTag(this.name());;
```

Comment:
extra `;`, unnecessary `this.`
```suggestion
    private final short shortValue = SAMTag.makeBinaryTag(name());
```